Updates based parallel gov-cloud topic (#5)
mulesoft/docs-eu-cloud: modules/ROOT/pages/runtime-configure-eu.adoc

> = Configure Mule runtime engine to Connect to the EU Control Plane

Reviewer: To avoid the whole sticky wicket of title capitalization and Mule runtime engine, I suggest using "Mule" alone in this title, since we spell it out directly in text thereafter.
```suggestion
= Configure Mule to Connect to the EU Control Plane
```

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> To use Mule runtime engine within the EU Control Plane, you must redefine the configuration properties that enable Mule runtime engine to point to the EU Control Plane. You must also provide the client ID and client secret of the environment belonging to your organization.

Reviewer: GLOBAL:
According to my MuleSoft terminology sources, neither "control plane" nor "runtime plane" are capitalized, because this is "capability terminology, not a product."
Please work with your team and PMM to get this resolved, if it continues to be an issue.

Reviewer: BTW, if we introduce "Mule runtime engine (Mule)" in that first line, we can use only "Mule" thereafter. Your choice.

Author: OK - thank you!

> By default, Mule runtime engine is configured to point to the US control plane. If you do not override these properties, the API Gateway capability of Mule runtime engine is disabled in the EU Control Plane.

Reviewer: Also per term resources, "API gateway"

Author: Got it - thanks!

> The properties you must override depend on the version of Mule runtime engine you are using, as described in the following sections.

Reviewer: ```suggestion
The properties you must override depend on the version of Mule runtime engine you are using: Mule 4.0 or Mule 3.x.
```
To promote independent, reusable topics, avoid referring to other "topics," "sections," or similar.
However, to set audience expectations for subtopics, try to mention them directly (not by title, but narratively).

== Configuring Mule 4.0

> 

Reviewer: GLOBALLY rework text to not have contiguous titles or headings. Insert something that, at a minimum, prepares the audience for the subtopics within a container.

Reviewer: Suggest something like...:

"You can configure Mule runtime engine (Mule) using `wrapper.conf`, the command line, or CloudHub."

If there is anything to say about under what circumstances a user might choose one method over another, suggest providing that here as well.

Author: Great - thank you!

=== wrapper.conf

> To configure Mule runtime engine using the `wrapper.conf` file, modify the following properties:

Reviewer: Suggestion:
```suggestion
To configure Mule runtime engine using the `wrapper.conf` file, modify the following properties, replacing `<n>` with an integer value that is unique within `wrapper.conf`:
```

wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
----

> In the previous example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.

Reviewer: See suggestion above.

If you keep the orig text, suggest using "In this example,..." because there is only one example in this subtopic.

Author: Updated this.

=== Command Line

> To configure Mule runtime engine from the command line, use the following options when starting Mule runtime engine. Note that `./` only applies to Unix environments.

Reviewer: Is it nec. to specify "...when starting the Mule runtime engine"? Does it change the meaning of the sentence to omit it (to configure, when starting)? It sounds a little odd, but that might be just the repetition of the full spelling.

Reviewer: Alternative:
```suggestion
To configure Mule runtime engine from the command line, use the following options (note that `./` applies only to Unix environments):
```

Author: Leaving as is.

anypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
----

== Configuring Mule 3.x

> 

Reviewer: DITTO re. transition sentence

Author: Got it - thanks!

=== wrapper.conf

> To configure `wrapper.conf` file, modify the following properties:

Reviewer: Consider making this sentence exactly the same as the one used in the same place for the Mule 4 text.
Use parallelism as much as possible.

Author: Got it - thank you!

----
wrapper.java.additional.<n>=-Danypoint.platform.client_id=<environment client ID>
wrapper.java.additional.<n>=-Danypoint.platform.client_secret=<environment client secret>
wrapper.java.additional.<n>=-Danypoint.platform.platform_base_uri=https://eu1.anypoint.mulesoft.com/apiplatform
wrapper.java.additional.<n>=-Danypoint.platform.coreservice_base_uri=https://eu1.anypoint.mulesoft.com/accounts
wrapper.java.additional.<n>=-Danypoint.platform.contracts_base_uri=https://eu1.anypoint.mulesoft.com/apigateway/ccs
wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
----

> In the previous example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.

Reviewer: DITTO edit suggestion for Mule 4

Author: Thanks!

=== Command Line

> To configure Mule runtime engine from the command line, use the following options when starting Mule runtime engine. Note that `./` only applies to Unix environments.

Reviewer: DITTO

Author: OK - thank you!

----
./mule -M-Danypoint.platform.client_id=<org client ID> -M-
Danypoint.platform.client_secret=<org client secret> -M-Danypoint.platform.platform_base_uri=https://eu1.anypoint.mulesoft.com/apiplatform -M-Danypoint.platform.coreservice_base_uri=https://eu1.anypoint.mulesoft.com/accounts -M-Danypoint.platform.contracts_base_uri=https://eu1.anypoint.mulesoft.com/apigateway/ccs -M-Danypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
----

=== CloudHub

> When deploying to CloudHub, you must add the following information and enable the autodiscovery feature in the Properties tab.

Reviewer: ```suggestion
When deploying to CloudHub, add the following information and enable the autodiscovery feature in the Properties tab:
```

Author: Leaving as is.

== See Also

* xref:index.adoc[EU Control Plane]

> * xref:platform-access-eu.adoc[To Access the EU Control Plane]

Reviewer: ```suggestion
* xref:platform-access-eu.adoc[Access the EU Control Plane]
```

Author: Got it - thank you!
